scripts/IBM Plex export FDK files.py

```diff
@@ -362,11 +359,8 @@ def checkCurveType(f):
 				if hasType == None:
 					typeDict[pType] = True
 
-	allTypes = typeDict.keys()
-	if 'curve' in allTypes:
-		return True
-	else:
-		return False
+	allTypes = typeDict
+	return bool('curve' in allTypes)
```

Comment:
You don't really need `bool()` here, the expression returns bool itself :)

Reply:
"Explicit is better than implicit."

Reply:
Removed the __bool()__